bin/createTestDB.js

```diff
@@ -49,8 +49,12 @@ const seed = {
   alpha: 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
 }
 
-// predictable simple semi psuedo random generator
-// https://stackoverflow.com/a/19303725/3785845
+/**
+ * predictable simple semi psuedo random generator (https://stackoverflow.com/a/19303725/3785845)
+ * @param seed the random seed
+ * @param m prand seed
```

Comment:
I'm an idiot - I need to pick a better const name for the default value of `m`.  i.e. DEFAULT_PRAND_BIAS - should probably then change the variable to `bias` also...